toolkit/tools/imager/imager.go

```diff
@@ -474,11 +474,19 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap m
 
 func configureDiskBootloader(systemConfig configuration.SystemConfig, installChroot *safechroot.Chroot, diskDevPath string, installMap map[string]string, encryptedRoot diskutils.EncryptedRootDevice) (err error) {
 	const rootMountPoint = "/"
+	const bootMountPoint = "/boot"
 
 	var rootDevice string
 
-	// Add bootloader
-	bootUUID, err := installutils.GetUUID(installMap[rootMountPoint])
+	// Add bootloader. Prefer a seperate boot partition if one exists.
+	bootDevice, ok := installMap[bootMountPoint]
+	bootPrefix := ""
+	if !ok {
+		bootDevice = installMap[rootMountPoint]
+		// If we do not have a seperate boot partition we will need to add a prefix to all paths used in the configs.
+		bootPrefix = "/boot"
+	}
```

Comment:
I'd also add a debug log saying what we've ended up going with. Unless that's covered in a different function.

Reply:
It gets logged at debug level when writing grub.cfg, which seems to be the pattern for many of these values.